MaterialDesignThemes.Wpf/Card.cs

```diff
@@ -5,15 +5,37 @@
 
 namespace MaterialDesignThemes.Wpf
 {
-    /// <summary>
-    /// A card is a content control, styled according to Material Design guidelines.
-    /// </summary>
     [TemplatePart(Name = ClipBorderPartName, Type = typeof(Border))]
     public class Card : ContentControl
     {
+        private Border _clipBorder;
+        private static readonly double DefaultUniformCornerRadius = 2.0;
```

Comment:
Any reason to not make this a `const`?

Reply:
No, i was just used to defaults being reference types. Const is certainly more appropriate.